Dockerfile

```diff
@@ -1,54 +1,32 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
 
 #MAINTAINER Cayde Dixon <me@cazzar.net>
 ENV PUID=1000 \
-    PGID=100  \
-    TargetFrameworkDirectory=/usr/lib/mono/
-
-RUN apt-get install apt
-RUN apt-get update && apt-get install -y gnupg curl wget ca-certificates apt-transport-https software-properties-common
-
-RUN curl https://bintray.com/user/downloadSubjectPublicKey?username=bintray | apt-key add -
-RUN curl https://mediaarea.net/repo/deb/debian/pubkey.gpg | apt-key add -
-RUN echo "deb http://dl.bintray.com/cazzar/shoko-deps jesse main" | tee -a /etc/apt/sources.list
-RUN echo "deb https://mediaarea.net/repo/deb/debian/ stretch main" | tee -a /etc/apt/sources.list
-RUN echo "deb http://ftp.debian.org/debian stretch-backports main" | tee -a /etc/apt/sources.list
-
-RUN apt-get update && apt-get install -y apt-utils libmediainfo0v5 librhash0 sqlite.interop jq unzip libunwind-dev && apt-get install -y -t stretch-backports gosu
-
-RUN curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel 2.2
+    PGID=100
 
 RUN mkdir -p /usr/src/app/source /usr/src/app/build
 COPY . /usr/src/app/source
 WORKDIR /usr/src/app/source
 
+RUN dotnet build -c=Release -o=/usr/src/app/build/ Shoko.CLI/Shoko.CLI.csproj
 
-#ADD https://github.com/NuGet/Home/releases/download/3.3/NuGet.exe .
-RUN nuget restore
-RUN msbuild /property:Configuration=CLI /property:OutDir=/usr/src/app/build/
-RUN rm /usr/src/app/build/System.Net.Http.dll
-
-FROM mono:6.0
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
 ENV PUID=1000 \
-    PGID=100 
-
-RUN apt-get install apt
-RUN apt-get update && apt-get install -y gnupg curl wget ca-certificates apt-transport-https software-properties-common
-
-RUN curl https://bintray.com/user/downloadSubjectPublicKey?username=bintray | apt-key add -
+    PGID=100 \
+    LANG=C.UTF-8 \
+    LC_CTYPE=C.UTF-8 \
+    LC_ALL=C.UTF-8
+    
 RUN curl https://mediaarea.net/repo/deb/debian/pubkey.gpg | apt-key add -
-RUN echo "deb http://dl.bintray.com/cazzar/shoko-deps jesse main" | tee -a /etc/apt/sources.list
-RUN echo "deb https://mediaarea.net/repo/deb/debian/ stretch main" | tee -a /etc/apt/sources.list
-RUN echo "deb http://ftp.debian.org/debian stretch-backports main" | tee -a /etc/apt/sources.list
+RUN echo "deb https://mediaarea.net/repo/deb/debian/ buster main" | tee -a /etc/apt/sources.list
 
-RUN apt-get update && apt-get install -y apt-utils libmediainfo0v5 librhash0 sqlite.interop jq unzip libunwind-dev && apt-get install -y -t stretch-backports gosu
+RUN apt-get update && apt-get install -y apt-utils gosu jq unzip libmediainfo0v5 librhash-dev libunwind-dev mono-runtime libmono-system-xml-linq4.0-cil
 
 WORKDIR /usr/src/app/build
 COPY --from=0 /usr/src/app/build .
 COPY ./dockerentry.sh /dockerentry.sh
 
 WORKDIR /usr/src/app/build/webui
-#RUN curl -L $(curl https://api.github.com/repos/ShokoAnime/ShokoServer-WebUI/releases | jq -r '. | map(select(.prerelease==false)) | .[0].assets[0].browser_download_url') -o latest.zip
```

Comment:
Don't delete this line.

Reply:
Sorry, I just rewrote the whole file and copy-pasted some things so I think I missed it